dataikuapi/dss/projectlibrary.py

```diff
@@ -0,0 +1,275 @@
+from ..utils import DataikuException
+
+
+class DSSLibrary(object):
+    """
+    A handle to manage the library items of a project
+    It saves locally a copy of taxonomy to help navigate in the library
+    All modifications done through this object and related library items are done locally and on remote.
+
+    Note: Taxonomy modifications done outside this library are not reflected locally.
+          You should reload the library in this case.
+    """
+    def __init__(self, client, project_key):
```

Comment:
I am not a fan of calling `reload` in the constructor.
For what I know, but I am not the biggest expert on the matter (You should ask guidance to Clement), usually in the client, the constructor will just init the handle, and do not perform any call. A subsequent call to `get_settings` will do it. cf what is done for the wiki.